docs/docs-beta/docs/dagster-plus/deployment/serverless/run-isolation.md

```diff
@@ -1,8 +1,73 @@
 ---
-title: "Serverless run isolation"
-displayed_sidebar: "dagsterPlus"
-sidebar_label: "Run isolation"
+title: 'Serverless run isolation'
+displayed_sidebar: 'dagsterPlus'
+sidebar_label: 'Run isolation'
 sidebar_position: 40
 ---
 
-# Serverless run isolation
\ No newline at end of file
+# Serverless run isolation
+
+Dagster+ Serverless offers two settings for run isolation: isolated and non-isolated. Isolated runs are the default and are intended for production, while non-isolated runs are intended for faster iteration during development.
+
+## What you'll learn
+
+- The differences between isolated runs and non-isolated runs
+- How to enable non-isolated runs
+- How to adjust your deployment settings to run multiple non-isolated runs concurrently
+
+---
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- An active Dagster+ Serverless deployment
+- An understanding of [Dagster+ deployment settings](/dagster-plus/deployment/settings)
+</details>
+
+---
+
+## Differences between isolated and non-isolated runs
+
+- [**Isolated runs**](#isolated-runs-default) execute in their own container. They're the default and are intended for production and compute-heavy use cases..
+- [**Non-isolated runs**](#non-isolated-runs) trade off isolation for speed. They must be launched manually and are intended for fast iteration during development.
+
+## Isolated runs (default)
+
+Isolated runs each take place in their own container with their own compute resources: 4 vCPUs and 16GB of RAM.
```

Comment:
Wonder if it's better to link to a `serverless` main page and list resource constraints there, in case we ever change them?

Reply:
The resource constraints appear a few times in this doc. I think it is fine to repeat them here. Can change in a follow up if you'd like